frontend/src/components/PageHeader.tsx

```diff
@@ -1,25 +1,70 @@
 "use client";
 
-import { useTranslation } from "react-i18next";
+import Lottie from "lottie-react";
+import { Trans, useTranslation } from "react-i18next";
 
+import { EMOJI_ASSETS } from "@/constants/assets";
 import { TRADE_SAFETY_NS } from "@/i18n";
 import { SafetyLevel } from "@/types";
 
 export interface PageHeaderProps {
   level: SafetyLevel;
   score: number;
+  lottieData: object | null;
 }
 
-export function PageHeader({ level, score }: PageHeaderProps) {
+const LEVEL_STYLES: Record<SafetyLevel, string> = {
+  danger: "font-bold text-error",
+  caution: "font-bold text-warning",
+  safe: "font-bold text-success",
+};
+
+export function PageHeader({ level, score, lottieData }: PageHeaderProps) {
   const { t } = useTranslation(TRADE_SAFETY_NS);
 
+  const assets = EMOJI_ASSETS[level];
+
   return (
-    <div className={"mb-4 rounded-lg"}>
-      <h1 className={"mb-2 text-3xl font-bold"}>
-        {t(`result.safetyLevel.${level}.title`)}
+    <div className="mb-8 flex flex-col items-center text-center">
+      <div className="mb-5 rounded-full bg-neutral/75 px-4 py-1.5">
+        <span className="text-sm font-medium text-neutral-content">
+          {t(`result.safetyLevel.${level}.badge`)}
+        </span>
+      </div>
+
+      <h1 className="mb-9 break-keep text-4xl font-bold leading-tight text-base-content">
+        <Trans
+          t={t}
+          i18nKey={`result.safetyLevel.${level}.title`}
+          components={[<strong key="0" className={LEVEL_STYLES[level]} />]}
+        />
       </h1>
-      <p className="mt-2 text-sm text-neutral">
-        {t(`result.safetyLevel.${level}.description`, { score })}
+
+      <div className="mb-9 flex size-32 items-center justify-center">
+        {lottieData ? (
+          <Lottie animationData={lottieData} loop autoplay />
+        ) : (
+          <picture>
+            <source srcSet={assets.webp} type="image/webp" />
+            <img
+              src={assets.gif}
+              alt={assets.alt}
+              className="size-full object-contain"
+            />
+          </picture>
+        )}
```

Comment:
```suggestion
        <LottiePlayer src={assets.lottie} autoplay loop className="size-full" />
```

장점:
- 코드 감소
- 선언적 & 유지보수 쉬움

Reply:
의도적으로 제외한 것이 아니라 제 부주의로 인해 누락했습니다.. 죄송합니다.

그런데 궁금한 것이 있습니다!
초기 gemini review 시 AbortController 개념에 대해 짚어주어서 코드에 적용을 했는데요. 
`LottiePlayer` 에 AbortController, Fallback 처리가 되어 있을까요?

comment에 fallback에 대한 질문이 포함되어 있어서 여쭙습니다!

Reply:
좋은 질문입니다!

LottiePlayer:
- AbortController: 없음
- Fallback: 없음

하지만 저는 Emoji의 경우 LottiePlayer 사용을 권장합니다 :)

이유:

1. 파일이 작아서 AbortController 없이도 충분합니다.
    - Lottie: 16-65 KB
    - 취소해도 절약: 미미

2. Fallback이 오히려 느립니다. 파일 크기:
    - WebP:   90-287 KB (6-12배 무거움)
    - GIF:    176-807 KB (11-19배 무거움)

미래에 대용량 Lottie를 사용할 땐 LottiePlayer에도 AbortController 지원을 추가해야겠네요!
어떻게 생각하시나요? @zhwltlr